eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Gov.hs

```diff
@@ -170,9 +177,15 @@ govTransition = do
   -- TODO Check the signatures
```

Comment:
I don't think anyone knows what this TODO refers to.. maybe we can remove it?

Reply:
@Soupstraw any ideas?

Reply:
I will make sure to remove the comment before merging if no one yells by tomorrow. :raised_hands:

Reply:
Thansk @Soupstraw ! :raised_hands: